src/app/components/algorithms/algorithm-view/algorithm-view.component.html

```diff
@@ -1,2 +1,44 @@
-<p>algorithm-view works!</p>
+<div *ngIf="algorithm" class="main-container">
+  <div class="header-container p-2">
+    <navigation-breadcrumb [links]="links"></navigation-breadcrumb>
+    <div>
+      <mat-chip-list>
+        <mat-chip *ngFor="let tag of testTags" [removable]="true" (removed)="removeTag(tag)">
+          {{tag}}
+          <mat-icon matChipRemove>cancel</mat-icon>
+        </mat-chip>
+        <mat-chip class="clickable-chip" (click)="addTag()">
+          <mat-icon>add</mat-icon>
+        </mat-chip>
+      </mat-chip-list>
+    </div>
+  </div>
 
+  <mat-tab-group>
+    <mat-tab [label]=tabOptions[0]>
```

Comment:
why are topOptions declared as a variable? They could just be inserted here as a string

Reply:
Done